packages/telemetry/test/unit/telemetryReporter.test.ts

```diff
@@ -199,17 +201,16 @@ describe('SpawnedTelemetryReporter', () => {
       kill: killStub
     });
     const options = { project, key };
-    const reporter = await SpawnedTelemetryReporter.create(options);
+    const reporter = await TelemetryReporter.create(options);
     reporter.sendTelemetryEvent('testName');
     expect(sendStub.calledOnce).to.be.true;
   });
 
   it('should not begin lifecycle when insights are disabled', async () => {
-    const env = new Env({});
```

Comment:
You can still have a separate test for the envar if you want. SFDX_DISABLE_TELEMETRY should still work.